sdk/eventhub/Azure.Messaging.EventHubs.Processor/src/EventProcessorClient.cs

```diff
@@ -244,16 +238,16 @@ public bool IsRunning
         public string Identifier { get; }
 
         /// <summary>
-        ///   The minimum amount of time to be elapsed between two load balancing verifications.
+        ///   The minimum amount of time for an ownership to be considered expired without further updates.
         /// </summary>
         ///
-        internal virtual TimeSpan LoadBalanceUpdate => TimeSpan.FromSeconds(10);
+        internal virtual TimeSpan OwnershipExpiration => TimeSpan.FromSeconds(30);
```

Comment:
The only intended point was to let the creator of the `PartitionLoadBalancer` decide on the expiration for ownership. I also wanted to keep the decision explicit for advanced users that want to implement their own Processor Client by accepting the expiration in its ctor. 

Do you think it would be better to add an additional ctor option on the `PartitionLoadBalancer` omitting this arg if implementors want a default?

Reply:
Making it visible for advanced users sounds like a reasonable argument to me. I'm fine with keeping it the way it is.